### PR TITLE
conn.h: fix ascii art

### PR DESCRIPTION
**Pull request by Lotterleben**
The little ASCII art in conn.h currently doesn't get displayed correctly in the HTMLized documentation (see http://doc.riot-os.org/group__net__conn.html), this PR should fix that :)

### CODE REVIEW

**Comment by Lotterleben** [comment]
Assigned Martine since she's part of docTF, feel free to hot potato of course

File: sys/include/net/conn.h [hunk 1]
```diff
@@ -15,6 +15,7 @@
  * About
  * =====
  *
+ * @code
```
Comment (miri64) [inline sys/include/net/conn.h]: could you use [markdown fences](https://www.stack.nl/~dimitri/doxygen/manual/markdown.html#md_fenced) instead, please?

Comment (Lotterleben) [inline sys/include/net/conn.h]: Oh ok sure, I used pktbuf.h as a reference (which uses `@code`as well), should I change that too while I'm at it?

Comment (miri64) [inline sys/include/net/conn.h]: Huh? Where? As far as I can see it uses both fencing and code-block by indentation. :-/

Comment (Lotterleben) [inline sys/include/net/conn.h]: You're right it does, just not on the outdated branch I looked at it on (weird sentence o.o). nevermind me then! (fences are installed in conn now, too)

**Comment by miri64** [comment]
ACK, please squash

**Comment by Lotterleben** [comment]
Squashed :)

**Pull request merged**